docs/app-directory/overview.md

```diff
@@ -7,18 +7,75 @@ hide_title: true
 
 # App Directory Overview
 
-The FDC3 App Directory provides trusted identity for financial desktop apps. This identity can be used to prevent spoofing and man-in-the-middle attacks when apps communicate with one another and exchange data. The App Directory also enables service discovery. Apps are registered with a declaration of the intents and context data that can be used when interoperating.
+An application directory (appD) is a structured repository of information about apps that can be used in an FDC3-enabled desktop. In other words, it’s a convenient way of storing and managing metadata about apps in your ecosystem. 
 
-## Core features
+The application metadata stored in appD includes the app name, type, details about how to run the application, its icons, publisher, support contact details and so on. All this information is readily available in one place and can be used both to populate a launcher or app catalog UI for your users, and by the desktop agent managing the apps on your desktop. In fact, if your smart desktop follows the FDC3 standard, appD is the primary way your desktop agent knows all the details about apps available to run on your desktop. Conversely, if an app is not listed in appD, the desktop agent can’t ensure its participation in context sharing or use it to resolve intents.
```

Comment:
Can we use "desktop agent" throughout? I get "smart desktop" but don't think it's the right term - seem to be a rather generic terms for lots of things out there...